delete-review-app/action.yml

```diff
@@ -0,0 +1,117 @@
+name: Delete Review App
+description: Delete Review App
+inputs:
+  azure-credentials:
+    description: A JSON string containing service principal credentials.
+    required: true
+  container-name:
+    description: azure container-name containing terraform state file
+    required: true
```

Comment:
container-name could default to 'terraform-state' and required: false

Reply:
changed